Area area
---
# Result
uint16 CODE_SUCCESS = 0
uint16 CODE_INVALID_AREA = 1
uint16 CODE_SAVE_FAILED = 2
uint16 CODE_UNKNOWN_ERROR = 99

uint16 code                # 0 is success, otherwise is error code
string message             # Information message
